sale_order_type_ux/views/sale_order_type_views.xml

```diff
@@ -16,7 +16,6 @@
         <field name="inherit_id" ref="sale_order_type.sot_sale_order_type_form_view"/>
         <field name="arch" type="xml">
             <field name="incoterm_id" position="after">
```

Comment:
@FedericoLaTorre thanks for fixing that bug. I think the position is better using in field `analytic_account_id` instead of `incoterm_id`